### PR TITLE
Update README (remove section header string literal

### PR DESCRIPTION
**Pull request by allaire**
It's not working anymore (probably a leftover in the doc)

### CODE REVIEW

**Comment by m-revetria** [comment]
Related to #994

File: README.md [hunk 1]
```diff
@@ -1021,8 +1021,6 @@ Look at this [issue](https://github.com/xmartlabs/Eureka/issues/96).
 * Set up a new header/footer data ....
 
 ```swift
-section.header = "Header Title" // use string literal as a header/footer data. HeaderFooterView conforms to ExpressibleByStringLiteral.
```
Comment (m-revetria) [inline README.md]: Actually you can create a `HeaderFooterView` instance from a string literal. So, maybe the deleted code can be replaced with:

```swift
let header: HeaderFooterView<UIView> = "Header Title" // use string literal as a header/footer data. HeaderFooterView conforms to ExpressibleByStringLiteral.
section.header = header
```

What do you think?

Comment (allaire) [inline README.md]: I think the exemple below is a better approach, and already documented:

```
section.header = HeaderFooterView(title: "Header title")
```

Comment (allaire) [inline README.md]: @m-revetria I can still add it if you prefer :)

Comment (m-revetria) [inline README.md]: @allaire sounds good to me. Thanks

**Pull request merged**